### PR TITLE
Fix test_extended.sh by setting openshift in PATH

### PR DESCRIPTION
**Pull request by akram**
@mfojtik could you please help me fixing these tests?
I want to add more tests here.

### CODE REVIEW

**Comment by mfojtik** [comment]
@akram @mnagy might assist here as he is also fixing extended tests now

File: hack/test-extended.sh [hunk 1]
```diff
@@ -144,12 +147,21 @@ wait_for_url "https://${OS_MASTER_ADDR}/api/v1beta1/minions/127.0.0.1" "" 0.25 8
 # Start the Docker registry (172.30.17.101:5000)
 start_docker_registry
 
-wait_for_command '[[ "$(osc get endpoints docker-registry -t "{{ if .endpoints}}{{ len .endpoints }}{{ else }}0{{ end }}" 2>/dev/null || echo "0")" != "0" ]]' $((5*TIME_MIN))
+# TODO: simplify when #4702 is fixed upstream
+wait_for_command '[[ "$(osc get endpoints docker-registry --output-version=v1beta1 -t "{{ if .endpoints }}{{ len .endpoints }}{{ else }}0{{ end }}" || echo "0")" != "0" ]]' $((5*TIME_MIN))
```
Comment (mnagy) [inline hack/test-extended.sh]: Why not use v1beta3 and .subsets instead of .endpoints?

File: hack/test-extended.sh [hunk 2]
```diff
@@ -59,7 +59,10 @@ start_server() {
     SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},${IP_ADDRESS}"
   done <<< "${ALL_IP_ADDRESSES}"
 
-  echo "[INFO] Create certificates for the OpenShift master"
+  GO_OUT="${OS_ROOT}/_output/local/go/bin"
+  # set path so OpenShift is available
+  export PATH=$GO_OUT:$PATH
+	echo "[INFO] Create certificates for the OpenShift master"
```
Comment (mnagy) [inline hack/test-extended.sh]: tab vs. spaces?

File: hack/test-extended.sh [hunk 3]
```diff
@@ -144,12 +147,21 @@ wait_for_url "https://${OS_MASTER_ADDR}/api/v1beta1/minions/127.0.0.1" "" 0.25 8
...
 echo "[INFO] Verifying the docker-registry is up at ${REGISTRY_ADDR}"
-wait_for_url_timed "http://${REGISTRY_ADDR}" "" $((2*TIME_MIN))
+wait_for_url_timed "http://${REGISTRY_ADDR}/v2/" "[INFO] Docker registry says: " $((2*TIME_MIN))
+
+#wait_for_command '[[ "$(osc get endpoints docker-registry -t "{{ if .endpoints}}{{ len .endpoints }}{{ else }}0{{ end }}" 2>/dev/null || echo "0")" != "0" ]]' $((5*TIME_MIN))
+
+#REGISTRY_ADDR=$(osc get --output-version=v1beta1 --template="{{ .portalIP }}:{{.port }}" \
+#  service docker-registry)
+#echo "[INFO] Verifying the docker-registry is up at ${REGISTRY_ADDR}"
+#wait_for_url_timed "http://${REGISTRY_ADDR}" "" $((2*TIME_MIN))
```
Comment (mnagy) [inline hack/test-extended.sh]: Maybe remove the commented bits?

File: hack/test-extended.sh [hunk 4]
```diff
@@ -59,7 +59,10 @@ start_server() {
     SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},${IP_ADDRESS}"
   done <<< "${ALL_IP_ADDRESSES}"
 
-  echo "[INFO] Create certificates for the OpenShift master"
+  GO_OUT="${OS_ROOT}/_output/local/go/bin"
+  # set path so OpenShift is available
+  export PATH=$GO_OUT:$PATH
```
Comment (mnagy) [inline hack/test-extended.sh]: Can we put the PATH setting somewhere up?

File: hack/test-extended.sh (hunk 1 above)
Comment (akram) [inline hack/test-extended.sh]: I would definitely agree, but not knowing about changes in v1beta3 I was not using it.
I am doing the changes now.

**Comment by danmcp** [comment]
@akram Bump

**Comment by mfojtik** [comment]
@danmcp @akram I think we can close this PR now as we completely refactored the extended tests in last 3 weeks and this patch is no longer valid.

**Comment by akram** [comment]
@mfojtik @danmcp agreed

**Pull request closed without merging**